package.json

```diff
@@ -2,9 +2,11 @@
   "name": "appium-adb",
   "version": "1.7.5",
   "description": "appium-adb",
-  "main": "lib/main.js",
+  "main": "./build/index.js",
   "scripts": {
-    "test": "./node_modules/.bin/mocha test/unit/*-specs.js"
+    "prepublish": "./node_modules/.bin/gulp prepublish",
```

Comment:
I was wrong with this `./node_modules/.bin` thing, npm already adds it to the path so `gulp prepublish` is the right way.